lib/load.php

```diff
@@ -104,11 +104,13 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/experimental/l10n.php';
 require __DIR__ . '/experimental/navigation-fallback.php';
 require __DIR__ . '/experimental/interactivity-api/script-loader.php';
+if ( gutenberg_is_experiment_enabled( 'gutenberg-no-tinymce' ) ) {
+	require __DIR__ . '/experimental/disable-tinymce.php';
+}
```

Comment:
Right now, this functionality is powered by a Gutenberg experiment. The experiment is supposed to work as a primary on/off button for the tinyMCE deprecation.